README.md

```diff
@@ -7,15 +7,15 @@ The repository to find popular Vale rules that are:
 
 ## Vale
 
-[Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important advantage of the software is its capability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/).
+[Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important feature of this tool is the ability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/). Vale demonstrates this capability within the [package hub](https://vale.sh/hub/) where you can find style guides prepared for [Google](https://vale.sh/hub/google/) and [Microsoft](https://vale.sh/hub/microsoft/).
 
-Many organizations that use this capability with Vale also publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
+Furthermore, organizations including [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their Vale configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many technical writers have re-iterated upon similar rules, aiming to deliver the same results.
```

Comment:
```suggestion
## Vale ecosystem and its limits

Furthermore, organizations including [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their Vale configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many technical writers have re-iterated upon similar rules, aiming to deliver the same results.
```

I think we need a section here. The title of the section can we rewritten.

I suggest this because this way, we would have a :

- What it's about
- the problems
- our solution

With current version:
- (many/lazy/fast-reading) people won't read the last sentences of vale section, while it's the most important.

And these last sentences are somehow opinionated, while the rest are descriptive, and factual.

Reply:
Good idea, but I'd prefer it to be a subsection instead. I'll update the README shortly, please review.